test/spec/oidc/renewTokensWithRefresh.ts

```diff
@@ -52,6 +51,10 @@ describe('renewTokensWithRefresh', function () {
     oauthUtil.loadWellKnownAndKeysCache(authInstance);
   });
 
+  afterEach(() => {
+    jest.spyOn(Date, 'now').mockRestore();
```

Comment:
this should happen automatically with the `restoreMocks` option set in jest.config. Was it not being restored?

Reply:
right, missed that one.